types/vote_set.go

```diff
@@ -560,6 +562,22 @@ func (voteSet *VoteSet) StringShort() string {
 		voteSet.height, voteSet.round, voteSet.signedMsgType, voteSet.maj23, frac, voteSet.votesBitArray, voteSet.peerMaj23s)
 }
 
+// LogString produces a logging suitable string representation of the
+// vote set.
+func (voteSet *VoteSet) LogString() string {
+	if voteSet == nil {
+		return nilVoteSetString
+	}
+	voteSet.mtx.Lock()
+	defer voteSet.mtx.Unlock()
+	voted, total, frac := voteSet.sumTotalFrac()
+
+	return fmt.Sprintf("Votes:%d/%d(%.3f) Validators:%q %v",
+		voted, total, frac,
+		voteSet.valSet,
+		voteSet.votesBitArray)
```

Comment:
not sure if this adds value in logs, does it? cc @alexanderbez

Reply:
Definitely! It was in the prototype "new" long message in the original ticket, but I'm down to remove it.

Reply:
I don't think it is super valuable. It is pretty fun to look at when watching logs, but not sure if thats a good reason to keep it

Reply:
great, I took it out :)